Pear/Controller/HomeViewController.swift

```diff
@@ -200,6 +218,12 @@ class HomeViewController: UIViewController {
             make.centerX.equalToSuperview()
         }
 
+        // TODO - remove later
```

Comment:
nitpick but remove comment as its already documented that the `feedbackButton` will be removed on line 33 and "remove later" is a bit vague